eclipselink/src/main/java/org/jipijapa/eclipselink/EclipseLinkPersistenceProviderAdaptor.java

```diff
@@ -69,7 +69,7 @@ public EclipseLinkPersistenceProviderAdaptor() {
     @SuppressWarnings({ "rawtypes", "unchecked" })
     @Override
     public void addProviderProperties(Map properties, PersistenceUnitMetadata pu) {
-        if (!properties.containsKey(ECLIPSELINK_TARGET_SERVER)) {
+        if (!properties.containsKey(ECLIPSELINK_TARGET_SERVER) && !pu.getProperties().containsKey(ECLIPSELINK_TARGET_SERVER)) {
```

Comment:
Could be changed to if (!pu.getProperties().containsKey(ECLIPSELINK_TARGET_SERVER)) {